frontend/src/components/Paper.svelte

```diff
@@ -1,28 +1,37 @@
 <script>
-  import { BreadcrumbItem, Button, TableBodyCell, TableBodyRow } from "flowbite-svelte";
+    import {BreadcrumbItem, Button, TableBodyCell, TableBodyRow} from "flowbite-svelte";
 
-  export let href;
-  export let title;
-  export let category;
-  export let slots;
-  export let show_category = false;
+    export let href;
+    export let title;
+    export let id;
+    export let category;
+    export let slots;
+    export let show_slots = false;
+    export let show_category = false;
 </script>
 
+
 <TableBodyRow>
-  <TableBodyCell>
-    <BreadcrumbItem>{title}</BreadcrumbItem>
-  </TableBodyCell>
-  {#if show_category}
-    <TableBodyCell>
-      <BreadcrumbItem href="/categories/{category.id}">{category.name}</BreadcrumbItem>
-    </TableBodyCell>
-  {/if}
-  {#if category.type === "Internal" }
-    <TableBodyCell>{slots}</TableBodyCell>
     <TableBodyCell>
-      <Button disabled={slots<=0} href={href} outline size="xs">
-        Start Review
-      </Button>
+        <BreadcrumbItem href="/categories/{category.id}/{id}">{title}</BreadcrumbItem>
```

Comment:
it uses its on id here, however in both cases Paper.svelte is used (in my-papers/+page.svelte and categories/[category_id]/+page.svelte) it is given no id therefore is always undefined